docs/home/api-info/webhooks.mdx

```diff
@@ -38,14 +39,18 @@ different messages when they become available - specifically:
 
 ## Authentication
 
-Validate requests to your public endpoint by comparing the webhook key with each request's HTTP header.
+When Metriport sends a webhook event, it includes an `x-metriport-signature` header. 
```

Comment:
nit: let's be consistent with terminology in technical docs - we don't refer to these as a webhook "event" in other places, let's use webhook "message" for consistency